mkdocs.yml

```diff
@@ -30,38 +33,56 @@ theme:
 repo_url: https://github.com/IDinsight/aaq-core
 
 nav:
-  - Overview:
+  - Welcome to AAQ:
       - What is AAQ: index.md
       - Usage: usage.md
+  - Core Components:
+      - components/index.md
+      - Admin App:
+          - components/admin-app/index.md
+          - Access Levels: components/admin-app/access-level.md
+          - Manage Content: components/admin-app/manage-content.md
+          - Playground: components/admin-app/playground.md
+      - Question-Answering Service:
+          - components/qa-service/index.md
+          - Semantic Search: components/qa-service/semantic-search.md
+          - LLM Response: components/qa-service/llm-response.md
+      - Triaging Service:
+          - components/triaging/index.md
+  - Other Components:
```

Comment:
Would it be easy to set up an index.md for Other components too?
Right now when I click this menu, I am directed to WhatsApp/setup.md which can be confusing!

I really like the card style display of features/components, so in this index.md we can have Whatsapp, Chat manager, Offline LLM cards?

Reply:
Done!